src/AutoRest.CSharp.V3/AutoRest.CSharp.V3.csproj

```diff
@@ -22,6 +23,23 @@
 
   <ItemGroup>
     <None Include="../../src/assets/**/*.cs" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="package.json" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="../../readme.md" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
+  <Target Name="NpmPackage" DependsOnTargets="Build">
+    <Exec Command="npm pack" WorkingDirectory="$(OutputPath)" ConsoleToMsBuild="true">
```

Comment:
What's the reason to put the `npm pack` logic here (baked into csproj) instead of being in the devops pipeline?

Reply:
Easier to find the output dll and put it into the root of the package.